lib/widgets/action_sheet.dart

```diff
@@ -358,15 +358,11 @@ void showTopicActionSheet(BuildContext context, {
       pageContext: context));
   }
 
-  if (optionButtons.isEmpty) {
-    // TODO(a11y): This case makes a no-op gesture handler; as a consequence,
-    //   we're presenting some UI (to people who use screen-reader software) as
-    //   though it offers a gesture interaction that it doesn't meaningfully
-    //   offer, which is confusing. The solution here is probably to remove this
-    //   is-empty case by having at least one button that's always present,
-    //   such as "copy link to topic".
-    return;
-  }
+  optionButtons.add(CopyTopicLinkButton(
+    narrow: TopicNarrow(channelId, topic,
+      // TODO(#1499): use the/a max message id of the topic
```

Comment:
Is there still a TODO here? A previous commit started using `lastOrNull` of a list of message IDs.